src/app/components/GalleryCard/GalleryCard.module.css

```diff
@@ -0,0 +1,36 @@
+.container {
+  display: grid;
+  grid-template-columns: 1fr 2fr;
+  background-color: var(--color-primary);
+  width: 85vw;
+  height: 25vh;
+  overflow: hidden;
+  margin: 1.5rem;
+  place-items: center;
```

Comment:
Does this the same as the line below, "align-items: center"?

Reply:
Thank you for the hint, I will try it out.